From 93ffa172802433a247c7ee2cf841033234a6803c Mon Sep 17 00:00:00 2001
From: Sailaja <sbandarupalli@habana.ai>
Date: Thu, 08 Oct 2020 08:31:25 +0300
Subject: [PATCH] [SW-20979] goya2: tstd: use un-calibrated coefficients

in goya2 asic, efuse values are not programmed with temperature
sensor calibration values. Need to use un-calibrated coefficeints

this patch can be reverted when efuse values are burnt

Change-Id: I8ce804384665ffd7227af28881c40a59e5939e7e
Signed-off-by: Sailaja <sbandarupalli@habana.ai>
---

diff --git a/preboot/thermal_v2.c b/preboot/thermal_v2.c
index 862fe6b..62aa3ce 100644
--- a/preboot/thermal_v2.c
+++ b/preboot/thermal_v2.c
@@ -89,13 +89,13 @@
 /* Hard coded each of the goya2 on-chip TS device properties */
 static struct hl_ts_cfg goya2_hl_ts_data[] = {
 	{.base = mmDCORE0_MME_TSTD_BASE, .alarm_a = 102, .alarm_b = 120,
-		.use_calib = true, .calib_offset = 10, .hl_tmp_data = { 0 }},
+		.use_calib = false, .calib_offset = 10, .hl_tmp_data = { 0 }},
 	{.base = mmDCON0_TSTD_BASE, .alarm_a = 102, .alarm_b =  120,
-		.use_calib = true, .calib_offset = 9, .hl_tmp_data = { 0 }},
+		.use_calib = false, .calib_offset = 9, .hl_tmp_data = { 0 }},
 	{.base = mmDCORE1_MME_TSTD_BASE, .alarm_a = 102, .alarm_b = 120,
-		.use_calib = true, .calib_offset = 8, .hl_tmp_data = { 0 }},
+		.use_calib = false, .calib_offset = 8, .hl_tmp_data = { 0 }},
 	{.base = mmDCON2_TSTD_BASE, .alarm_a = 102, .alarm_b = 120,
-		.use_calib = true, .calib_offset = 7, .hl_tmp_data = { 0 }}
+		.use_calib = false, .calib_offset = 7, .hl_tmp_data = { 0 }}
 };
 
 void hl_ts_data_init(void)
